Dockerfile

```diff
@@ -7,7 +7,7 @@
 #
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
-FROM quay.io/eclipse/che-sidecar-kubernetes-tooling:1.0.0
+FROM quay.io/eclipse/che-sidecar-kubernetes-tooling:1.0.4
```

Comment:
I think this version should be fixed (i.e. `1.0.4-a46d022`). Since these images are built off commits -- e.g. tag `0.0.10-70f9d17` -- depending on a rolling tag here does not make sense. 

#### Currently
1. Kubernetes tooling image is updated, new image pushed onto 1.0.4 tag
2. Build of che-sidecar-camelk has to be manually (?) triggered to take changes in kubernetes-tooling image
3. This build results in a *new* image for e.g. `che-sidecar-camelk:0.0.10-70f9d17`, even though the commit hash suggests it's immutable.
4. It's also hard to track when this image needs to be rebuilt -- if there was a new kubernetes tooling image two weeks ago, you have to compare timestamps to check whether the current camelk image has the correct updates.

#### Suggestion
- Depend on `kubernetes-tooling:1.0.4-a46d022`
- If kubernetes-tooling image is updated, issue a commit bumping the hash here, triggering a new image
- From quay.io, it's easy to find which k8s-tooling image was used in the build, since you can follow the commit hash in the tagged image.

Reply:
@benoitf WDYT?

Reply:
@amisevsk yes I agree 👍

Reply:
updated